core/validatorapi/validatorapi.go

```diff
@@ -553,8 +553,17 @@ func (c Component) submitRegistration(ctx context.Context, registration *eth2api
 
 // SubmitValidatorRegistrations receives the partially signed validator (builder) registration.
 func (c Component) SubmitValidatorRegistrations(ctx context.Context, registrations []*eth2api.VersionedSignedValidatorRegistration) error {
+	if len(registrations) == 0 {
+		return nil // Nothing to do
+	}
+
+	slot, err := c.slotFromTimestamp(ctx, time.Now())
```

Comment:
what happens when the machine's time is not in sync with actual time?

Reply:
there is no such thing as actual time in distributed system